live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/repository/mapper/property/PropertiesMapper.java

```diff
@@ -57,31 +57,31 @@
 @RequiredArgsConstructor
 public class PropertiesMapper
 {
-    public Set<CustomPropertyDelta<?>> calculateCustomPropertiesDelta(RepoEvent<DataAttributes<NodeResource>> event)
+    public Set<PropertyDelta<?>> mapProperties(RepoEvent<DataAttributes<NodeResource>> event)
```

Comment:
Very minor - but maybe a small name change? I don't mind that name staying.
```suggestion
    public Set<PropertyDelta<?>> mapPropertiesDelta(RepoEvent<DataAttributes<NodeResource>> event)
```